pkgdb2/api/extras.py

```diff
@@ -262,11 +275,14 @@ def api_notify():
     :kwarg eol: Set to True if you want to include end of life
         distributions
     :kwarg format: Specify if the output if text or json.
+    :kwarg namespace: Specify if the namespace should be included in the
+        output (defaults to False).
```

Comment:
Because that would change the output and thus break tools relying on this format